### DISCUSSION TITLE
How to add the Ubuntu image's running user to groups?

### DISCUSSION THREAD

**Discussion by Avasam** [actions/virtual-environments#5981] [Image configuration]
During install, the python module `keyboard` runs `dumpkeys` to extract all key names.  
This fails unless the user running the command is in the `input` and `tty` groups.  
https://github.com/Toufool/Auto-Split/runs/7551506003?check_suite_focus=true#step:6:54

I cannot simply run `sudo usermod -a -G tty,input $USER`  as the groups aren't updated for the current session.  
I cannot run pip as root (`sudo pip install ...`) because this messes up packages installs in a way that breaks both `Pyright` and `PyInstaller`.

Is there any solution?

**Comment by al-cheb** [comment]
```
- run: |
      sudo usermod -aG tty,input $USER
      sudo -s -u ${USER} bash -c 'whoami && groups'
```

![image](https://user-images.githubusercontent.com/47745270/181439526-b34aa18d-f0c3-4e89-aa40-0b46c9d656ef.png)
![image](https://user-images.githubusercontent.com/47745270/181574703-86933c6c-c7f7-4734-aad2-7d04b8f5393d.png)

**Reply by Avasam** [reply]
Thanks! I did not know about the user `-u, --user` flag!
